docs/modules/ROOT/pages/project-creator.adoc

```diff
@@ -0,0 +1,110 @@
+
+* config
+* cluster_name
+* cluster_discovery_token
+* cluster_address
+* ssl_key_path
+* ssl_cert_path
+* ssl_key_password
+* cluster_api_base
+
+are the ones that are used in this template. More placeholders are listed in link:clc-project.adoc[project creator reference documentation].
+
+To create a project, run the following command:
+
+[source,bash]
+----
+clc -c <your-config> project create go-client-getting-started
+----
+
+The project will be created in the current working directory. If you want it to be created in a different directory, you can use `--output-dir` flag. After creating the project, you can directly run it without any effort, because placeholders already configured the project based on the given configuration.
+
+== Creating and Using Your Own Templates
+
+It is also possible users to create their own templates and use them. While creating your own template the following rules should be kept in mind:
+
+* Templates are in link:https://pkg.go.dev/text/template[Go template] format.
+* You can create a "template.yaml" file in template's root directory and add it a "defaults" node to add default values.
+* Template files must have the ".template" extension.
+* Files with "." and "_" prefixes are ignored unless they have the ".keep" extension.
+* All files with ".keep" extension are copied by stripping the ".keep" extension.
+* Other files are copied verbatim.
+*Properties are read from the following resources in order:
++
+1. defaults node of `templates.yaml` (keys cannot contain punctuation)
+2. `config.yaml`
+3. User passed key-values in the "KEY=VALUE" format. The keys can only contain lowercase letters, numbers and the underscore character.
+
+After creating your own template, you need to push it into a Github repository. That repository should be in a separate organization, so it is better to create an account/organization for your templates.
+
+To list them, you can use the `template list` command with `--source` flag:
+
+[source,bash]
+----
+clc template list --source=<your-organization-name>
+----
+
+To create a project from your template, run the following command:
+
+[source,bash]
+----
+clc -c <your-config> project create <your-template> --source=<your-organization-name>
+----
+
+You can also use any placeholder you want in your templates, and pass them with the create command:
+
+[source,bash]
+----
+clc -c <your-config> project create <your-template> foo=bar --source=<your-organization-name>
+----
+
+`foo` placeholder in your template will be replaced with `bar` after project is created.
+
+== Conclusion
+
+In this guide, you learned how to use default Hazelcast templates, and moreover how to create your own templates and use them to create projects. This is a powerful tool that creates projects for you just using a single command, and it is ready to run.
+
+Be sure to check out the link:clc-template.adoc[template] and link:clc-project.adoc[project] commands' references for all commands.
```

Comment:
For full details of the usage and parameters of the `clc template` and `clc project` commands, see the link:clc-template.adoc[clc template] and link:clc-project.adoc[clc project] documentation.